config/routes.rb

```diff
@@ -43,7 +43,7 @@ def matches?(request)
         # this to be accessible when using other authenticators to login.
         put  '/:authenticator/:account/api_key'  => 'credentials#rotate_api_key'
 
-        post '/authn-k8s/:service_id/inject_client_cert' => 'authenticate#k8s_inject_client_cert'
+        post '/authn-k8s/:service_id/inject_client_cert' => 'authenticate#k8s_inject_client_cert'  # <-----------------------------------
```

Comment:
what are all these `# <--------------`?

Reply:
Yeah, I was using these just to mark important things for myself. I'm deleting these when I submit the PR as a non-WIP